modules/nw-multus-add-pod.adoc

```diff
@@ -31,57 +50,57 @@ endif::sriov[]
 
 .Procedure
 
-To add a Pod with additional networks, complete the following steps:
+. Add an annotation to the Pod resource definition. Only one of the following annotation formats can be used.
 
-. Create the Pod resource definition and add the `k8s.v1.cni.cncf.io/networks`
-parameter to the Pod `metadata` mapping. The `k8s.v1.cni.cncf.io/networks`
-accepts a comma separated string of one or more NetworkAttachmentDefinition Custom Resource (CR) names:
+.. To attach an additional network without any customization, add an annotation with the following format. Replace `<network>` with the name of the additional network to associate with the Pod.
 +
 [source,yaml]
 ----
 metadata:
   annotations:
     k8s.v1.cni.cncf.io/networks: <network>[,<network>,...] <1>
 ----
-<1> Replace `<network>` with the name of the additional network to associate
-with the Pod. To specify more than one additional network, separate each network
+<1> To specify more than one additional network, separate each network
 with a comma. Do not include whitespace between the comma. If you specify
 the same additional network multiple times, that Pod will have multiple network
 interfaces attached to that network.
-+
-In the following example, two additional networks are attached to the Pod:
+
+.. To attach an additional network with customizations, add an annotation with the following format.
 +
 [source,yaml]
 ----
-apiVersion: v1
-kind: Pod
 metadata:
-  name: example-pod
   annotations:
-    k8s.v1.cni.cncf.io/networks: net1,net2
-spec:
-  containers:
-  - name: example-pod
-    command: ["/bin/bash", "-c", "sleep 2000000000000"]
-    image: centos/tools
+    k8s.v1.cni.cncf.io/networks: |-
+      [
+        "name": "<network>", <1>
+        "namespace": "<namespace>", <2>
+        "default-route": ["<default-route>"] <3>
+      ]
 ----
+<1> Specify the name of the additional network defined by an NetworkAttachmentDefinition CR.
+<2> Specify the namespace where the NetworkAttachmentDefinition CR is defined.
+<3> Optional: Specify an override for the default route, such as `192.168.17.1`.
+
 
-. Create the Pod by running the following command:
+. To create the Pod, enter the following command:
 +
+[source,terminal]
 ----
 $ oc create -f pod.yaml
 ----
 
-. Optional: Confirm that the annotation exists in the Pod CR by running the
-following command. Replace `<name>` with the name of the Pod.
+. Optional: To Confirm that the annotation exists in the Pod CR, enter the following command. Replace `<name>` with the name of the Pod.
```

Comment:
Might help readability as:
_...enter the following command, replacing `<name>` with..._

Reply:
@bobfuru, but does it feel more like a run-on sentence? I wanted to avoid abusing commas.

Reply:
Fair point!